### PR TITLE
Fix flashing delete modal

### LINKED ISSUE #85: When trying te delete a backup, the backup modal appears and disappears in sequence, not allowing a click
Hi,

Thanks for your package! 

I have installed it, and everything works as advertized, except the deletion of backups. When clicking on the delete button, a confirmation modal pops up, and is the closed then again in sequence with the xhr-call in the background. Clicking the confirmation button does not do anything. The strange thing is that all this is working correctly on the test environment (homestead). Update: I was wrong, this is happening on both sites, both homestead and lamp.

But this problem occurring on the staging environment (identical to the live environment), which is an Lamp-environment (ubuntu 18.04/apache2/php7.4.

I have updated laravel and nova to the latest versions, and republished the nova assets. Is there something that I am missing here?

### PR DESCRIPTION
**Pull request by quintenbuis**
This fixes #85

Only downside it only really works when polling is set to a minimum of 2 in the config. When set to 1 it will flash for 1 / 2 times and then it stops. Any ideas how to fix this will be welcome!

### CODE REVIEW

**Comment by quintenbuis** [comment]
Fixed issue with polling. Atleast when you don't have devtools open..

File: resources/js/components/BackupTool.vue [hunk 1]
```diff
@@ -131,15 +133,21 @@ export default {
...
+                    }
                 }, Nova.config.nova_backup_tool.polling_interval * 1000);
 
                 this.$once('hook:beforeDestroy', () => {
                     window.clearInterval(poller);
                 });
             }
         },
+
+        toggleModal({ state }) {
```
Comment (freekmurze) [inline resources/js/components/BackupTool.vue:148]: I think `setModalVisibility` would be a better now, as you're not really toggling.

**Review by freekmurze — CHANGES_REQUESTED** [review]

File: resources/js/components/BackupTool.vue [hunk 2]
```diff
@@ -131,15 +133,21 @@ export default {
...
+                    }
                 }, Nova.config.nova_backup_tool.polling_interval * 1000);
 
                 this.$once('hook:beforeDestroy', () => {
                     window.clearInterval(poller);
                 });
             }
         },
+
+        setModalVisibility({ state }) {
```
Comment (freekmurze) [inline resources/js/components/BackupTool.vue:148]: Instead of passing an object, passing a boolean would better simpler/better.

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**